### ISSUE TITLE
is `time` a reserved / special variable name?

### ISSUE THREAD

**Issue by rlisahuang** [UCSD-PL/vscode#53]
The code below breaks the update of PB (see the orange dots). The output panel shows a weird error message. 

```py
import matplotlib.pyplot as plt
import pandas as pd
import io


csv = '''
ID,group,time,success
P1,control,18.6,1
P2,control,15.42,1
P3,control,25.55,0
P4,control,12.56,0
P5,control,8.67,1
P6,experiment,7.31,0
P7,experiment,9.66,0
P8,experiment,13.64,1
P9,experiment,14.92,1
P10,experiment,18.47,1
'''


df = pd.read_csv(io.StringIO(csv))

# create a box plot where each box refers to a group
df.boxplot(column='time', by='group')

# remove the grid and increase the limit of y axis to 30
plt.grid(False)
plt.ylim(0, 30)

# add a dashed horizontal line at y=25
plt.axhline(y=25, linestyle='--')

# add a label "limit" next to the horizontal line
plt.text(x=2.45, y=25.5, s='limit')

# for each group, add scatter dots on top of the box using the time data, and mark the dots with blue if success=1, otherwise red
for grp in df.group.unique():
    success = df[df.group == grp].success
    time = df[df.group == grp].time
    plt.scatter (x=grp*len (success), y=time, c=success, s=100, marker= 'o', cmap= 'RdYlGn' )

```
![Screenshot 2023-03-13 at 10 27 14 PM](https://user-images.githubusercontent.com/29112035/224905542-75ffcf58-c2f1-469d-bd8e-c098f6f2ae29.png)

After tweaking and experimenting with the code, I realize that the variable name `time` is the source of the issue. I cannot simply have a variable named `time` because we have the following line in `run.py`:
```
 line 431, in adjust_to_next_time_step
    next_time = env["time"]+1
```

Thoughts on resolving this? @slerner

**Comment by KasraF** [comment]
I can't believe we haven't run into this before! Instead of using hardcoded variable names, we could generate a UUID (or some other random string) and assign that to a const to use instead? i.e.
```py
TIME = str(uuid.uuid1())
# ...
next_time = env[TIME] + 1
```

**Comment by rlisahuang** [comment]
Looks reasonable to me!

**Comment by KasraF** [comment]
Fixed in 8f8011d2acc296502a388b1207bd2d07563e05bd. 

I had to use _another_ hardcoded variable name unfortunately, since [`RTVDisplay.ts`](https://github.com/UCSD-PL/vscode/blob/leap/src/vs/editor/contrib/rtv/browser/RTVDisplay.ts#L58) needs to be aware of their names. But if a user wants to use the variable name `_projection_boxes_time`... at that point that's really their own responsibility :grin:

I'm keeping this open for now in case the fix introduced new bugs, but @rlisahuang please feel free to close if you're happy with the fix.
